extensions/dependencies/code_repl_prediction.html

```diff
@@ -1,3 +1,4 @@
+<script src="/templates/dev/head/domain/classifier/PredictionResultObjectFactory.js"></script>
```

Comment:
Just wondering, for this file and the next few ones -- why do we need this change?

Reply:
Since previously we script-imported these files in the pages where we used the extensions (CodeRepl) now that webpack is used in `/templates/dev/head` we need to script-import them here.

I think the question behind this is also why we depend on files from `/templates/dev/head` when `/extensions` should be "standalone" and not depended on the main parts of the code.

Reply:
I think this is a longer discussion about what "extensions" is and what the boundary is between "core" and "extensions". Can we talk about that in our next meeting?